homeassistant/components/aten_pe/translations/de.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Ger\u00e4t ist bereits konfiguriert"
+        },
+        "error": {
+            "cannot_connect": "Verbindung fehlgeschlagen, versuchen Sie es erneut",
+            "unknown": "Unerwarteter Fehler"
+        },
+        "step": {
+            "user": {
```

Comment:
Adding a `title` is required. Adding a `description` is highly useful.

Reply:
Isn't `"title": "Zugangsdaten"` what you're referring to?

Reply:
Can skip the title and just use the one of the integration itself